nifi-extension-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-client-service/src/main/java/org/apache/nifi/elasticsearch/ElasticSearchClientServiceImpl.java

```diff
@@ -1070,4 +1070,14 @@ private Response performRequest(final String method, final String endpoint, fina
 
         return client.performRequest(request);
     }
+
+    private String readContentAsUtf8String(HttpEntity entity) throws UnsupportedOperationException, IOException {
+        return this.readContentAsString(entity, StandardCharsets.UTF_8);
+    }
+
+    private String readContentAsString(HttpEntity entity, Charset charset) throws UnsupportedOperationException, IOException {
+        try (InputStream responseStream = entity.getContent()) {
+            return IOUtils.toString(responseStream, charset);
+        }
+    }
```

Comment:
No need to use `IOUtils`, you can use a `String` constructor and then drop the need to include in the method signatures `UnsupportedOperationException`. I also believe you can them remove the import for `IOUtils`.
```suggestion
    private String readContentAsUtf8String(HttpEntity entity) throws  IOException {
        return this.readContentAsString(entity, StandardCharsets.UTF_8);
    }

    private String readContentAsString(HttpEntity entity, Charset charset) throws  IOException {
        try (InputStream responseStream = entity.getContent()) {
            return new String(responseStream.readAllBytes(), charset);
        }
    }
```

Reply:
Thanks @dan-s1. Fixed.